frontend/styles/var/breakpoints.scss

```diff
@@ -1,12 +1,6 @@
-$break-xxl: 1600px;
-$break-xl: 1500px;
-$break-lg: 1400px;
-$break-md: 990px;
-$break-sm: 880px;
-$break-xs: 768px;
-$break-mobile-lg: 650px;
-$break-mobile-md: 576px;
-$break-mobile-sm: 480px;
-$break-mobile-xs: 320px;
-$tooltip-break-md: 1000px; // Prevents horizontal scrolling off viewport
-$table-controls-break: 1150px;
+$break-xl: 1440px;
+$break-lg: 1280px;
+$break-md: 1024px;
+$break-sm: 768px;
+$break-xs: 480px;
+$break-xxs: 320px;
```

Comment:
Can you link to Figma/issue # for these new breakpoints

Reply:
No Figma yet, this is currently in drafting and I'm acting as the product designer. The story is here: https://github.com/fleetdm/fleet/issues/25791